src/components/StakingCommunityCallout.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import { useIntl } from "gatsby-plugin-intl"
+import styled from "styled-components"
+import { graphql, useStaticQuery } from "gatsby"
+import { getImage } from "gatsby-plugin-image"
+
+import ButtonLink from "./ButtonLink"
+import CalloutBanner from "./CalloutBanner"
+
+import { translateMessageId } from "../utils/translations"
+import { trackCustomEvent } from "../utils/matomo"
+
+const StyledCallout = styled(CalloutBanner)`
+  margin: 4rem 0;
+`
+
+const ButtonContaier = styled.div`
+  display: flex;
+  gap: 1rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+  }
+`
+
+const StyledButtonLink = styled(ButtonLink)`
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: 100%;
+  }
+`
+
+const StakingCommunityCallout = ({ className, id }) => {
+  const intl = useIntl()
+  const { image } = useStaticQuery(graphql`
+    {
+      image: file(relativePath: { eq: "enterprise-eth.png" }) {
+        childImageSharp {
+          gatsbyImageData(
+            width: 500
+            layout: CONSTRAINED
+            placeholder: BLURRED
+            quality: 100
+          )
+        }
+      }
+    }
+  `)
+
+  return (
+    <StyledCallout
+      image={getImage(image)}
+      alt={translateMessageId("page-staking-image-alt", intl)}
+      titleKey={"page-staking-join-community"}
+      descriptionKey={"page-staking-join-community-desc"}
+      id={id}
+      className={className}
+    >
+      <ButtonContaier>
```

Comment:
Not a big fan of these line-wrapping button styles:

![Image 2022-04-06 at 9 11 09 PM](https://user-images.githubusercontent.com/8097623/162118563-a56e38ee-0862-4021-8987-1f80615e0aba.jpg)

I'd suggest:
- A) drop the verbs (e.g. so just "Discord" instead of "Join Discord"
- B) Use icons instead (like we do for the footer social links)

Reply:
I think some stats & social validation would be good here as well, e.g. "Join a community of thousands of Ethereum stakers around the globe".

Reply:
41.4k members according to their subreddit: https://www.reddit.com/r/ethstaker/